devops/roles/analytics-druid/tasks/main.yml

```diff
@@ -0,0 +1,7 @@
+# include deploy and configuration for druid role
+- include: install.yml
+  tags: install
+- include: deploy.yml
+  tags: deploy
+- include: test.yml
+  tags: test
```

Comment:
Installation and deployment should be differnt roles, as installtion is one time and won't change a lot. But deployment is acviely developed. And it's not advised not to keep tags in roles.

Reply:
@rjshrjndrn Yes, this has not been worded properly. We first provision the components and then start the different services.  We will move everything into the install.yml file.

Reply:
Does that mean different roles for installation and deployment? sounds cool.